Akkadu/rsi-widgets
# Update docs and add Live API Key and Room testing demo

# Update docs and add Live API Key and Room testing demo
The purpose of this PR is to improve the docs of our RSI SDK and fix any bugs that will popup during installation or using the SDK, for more info check the Notion pages below. 

https://www.notion.so/akkadu/Copy-of-Multilingual-SDK-Improve-Documentation-and-Github-repos-7168fcc1595a4a32b40cb25fd0600023

https://www.notion.so/akkadu/Copy-of-To-integrate-the-RSI-SDK-on-a-page-with-all-flavors-React-Vue-Vanilla-and-experience-how--1ffe255f713b46d78d9b17ddb041204d

## How To Test
1. Make sure that all the changes in the Notion task are there in the deploy preview. 
2. Make sure that the [react example demo deploy preview](https://deploy-preview-43--rsi-akkadu-demo.netlify.app/) is working properly, try using all the links on homepage. (The deploy preview may not work because of not being whitlisted :( ) but you can test locally in that case, need to make another PR to whitelist the deploy preview and whitelist react demo.





## Screenshots (if appropriate)

## Checklist



加油！
